dataManipulation/generarTablasIngresos.py

```diff
@@ -69,28 +67,26 @@ def transform(df,verbose=True):  #all of this can be written more compactly with
     hdia=(df.tipo=='h.dia')
     nbinj=(df.newborn_injury==1)
     
-    print('total plancms events ',sum(plancms))
-    print('total urgcms events ',sum(~plancms))
-    print('total hdia events ',sum(hdia))
-    
     #auxiliary cols
     df['nbinj']=df['newborn_injury']#shorter name
-    df['tot']=df.groupby(['id']).ing.transform(sum)
+
     #Describing all the characteristics of each episode
     #And computing the number of similar episodes per id using groupby and transform(sum)
     ###################################################
-    df['urg']=np.where((~hdia) & (urg), 1,0)
-    df['prog']=np.where((~hdia) & (~urg), 1,0)
+    
+    #PRIORITY ACCORDING TO OSAKIDETZA ADMINISTRATIVE CRITERIA
+    df['urg']=np.where( (urg), 1,0)
+    df['prog']=np.where( (~urg), 1,0)
     df['urg_num']=df.groupby(['id'])['urg'].transform(sum)
     df['prog_num']=df.groupby(['id'])['prog'].transform(sum)
     
-    df['urgcms']=np.where((~hdia) & (~plancms), 1,0)
```

Comment:
This was wrongfully excluding hdia patients!! That step is and should be carried by getData